pulsar-broker/src/main/java/org/apache/pulsar/broker/intercept/BrokerInterceptors.java

```diff
@@ -93,107 +97,143 @@ public static BrokerInterceptor load(ServiceConfiguration conf) throws IOExcepti
         }
```

Comment:
Looks like we don't need to return DISABLED here?

Reply:
Yes, this method will return DISABLE at last if no interceptors are configured